code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -577,19 +580,10 @@
 			break
 	return ..()
 
-/obj/item/circuitboard/machine/vending/donksofttoyvendor
-	name = "Donksoft Toy Vendor (Machine Board)"
-	build_path = /obj/machinery/vending/donksofttoyvendor
-	req_components = list(
-		/obj/item/stack/sheet/glass = 1,
-		/obj/item/vending_refill/donksoft = 1)
-
-/obj/item/circuitboard/machine/vending/syndicatedonksofttoyvendor
-	name = "Syndicate Donksoft Toy Vendor (Machine Board)"
-	build_path = /obj/machinery/vending/toyliberationstation
-	req_components = list(
-		/obj/item/stack/sheet/glass = 1,
-		/obj/item/vending_refill/donksoft = 1)
+///Used to print donksoft toy vendors through illegal tech.
+/obj/item/circuitboard/machine/vendor/donksofttoyvendor/Initialize(mapload)
+	. = ..()
+	set_type(/obj/machinery/vending/donksofttoyvendor)
```

Comment:
What's the point of this if you can screwdriver any board into a donksoft toy vendor?

Reply:
Just so people can still print it through illegal tech, I didn't want to remove it because screwdriver'ing boards isn't known by everyone and I didn't want to hide building the vendor behind knowing it.

Reply:
So it allows you to bypass needing illegal tech for donksofts by knowing how to screwdriver a vendor board? 😕

Reply:
Its still a vendor board so it should be possible, which is what I meant when I said I was making it consistent with the rest of the vendors.

Reply:
:-1: Repathing good but makes it too easy to make a Donksoft.

Reply:
I don't see how its a problem, the refill canisters are already rare/hard to get, the boards for them should be achieved the same as any other vendor, rather than **only** being locked behind 3 toxin experiments because _someone_ decided to make it a subtype of something that doesn't exist.